# HG changeset patch
# User Sahil Badyal <sbadyals@gmail.com>
# Date 1424969154 -19800
#      Thu Feb 26 22:15:54 2015 +0530
# Node ID 6065bd58db2be0a849d9bcec621dccf64ac6fdd1
# Parent  30eefd9ddb48071b40271262d16a4f926fae42e6
use contribute/donate consistently in GUI (bug #44335)

* main-window.h, main-window.cc (main_window::open_contribute_page):
Link to contribute.html instead of donate.html
(main_window::open_donate_page): New function.

diff --git a/libgui/src/main-window.cc b/libgui/src/main-window.cc
--- a/libgui/src/main-window.cc
+++ b/libgui/src/main-window.cc
@@ -631,23 +631,23 @@ void
 main_window::open_agora_page (void)
 {
   QDesktopServices::openUrl (QUrl ("http://agora.octave.org"));
 }
 
 void
 main_window::open_contribute_page (void)
 {
-  QDesktopServices::openUrl (QUrl ("http://octave.org/donate.html"));
+  QDesktopServices::openUrl (QUrl ("http://octave.org/contribute.html"));
 }
 
 void
-main_window::open_developer_page (void)
+main_window::open_donate_page (void)
 {
-  QDesktopServices::openUrl (QUrl ("http://octave.org/get-involved.html"));
+  QDesktopServices::openUrl (QUrl ("http://octave.org/donate.html"));
 }
 
 void
 main_window::process_settings_dialog_request (const QString& desired_tab)
 {
   if (_settings_dlg)  // _settings_dlg is a guarded pointer!
     {                 // here the dialog is still open and called once again
       if (! desired_tab.isEmpty ())
@@ -1990,20 +1990,20 @@ main_window::construct_help_menu (QMenuB
 
   _octave_packages_action = add_action (help_menu, QIcon (),
             tr ("Octave Packages"), SLOT (open_octave_packages_page ()));
 
   _agora_action = add_action (help_menu, QIcon (),
             tr ("Share Code"), SLOT (open_agora_page ()));
 
   _contribute_action = add_action (help_menu, QIcon (),
-            tr ("Contribute to Octave"), SLOT (open_contribute_page ()));
+            tr ("Contribute"), SLOT (open_contribute_page ()));
 
   _developer_action = add_action (help_menu, QIcon (),
-            tr ("Octave Developer Resources"), SLOT (open_developer_page ()));
+            tr ("Donate to Octave"), SLOT (open_donate_page ()));
 
   help_menu->addSeparator ();
 
   _about_octave_action = add_action (help_menu, QIcon (),
             tr ("About Octave"), SLOT (show_about_octave ()));
 }
 
 void
diff --git a/libgui/src/main-window.h b/libgui/src/main-window.h
--- a/libgui/src/main-window.h
+++ b/libgui/src/main-window.h
@@ -128,17 +128,17 @@ public slots:
   void open_online_documentation_page (void);
   void display_release_notes (void);
   void load_and_display_community_news (int serial = -1);
   void display_community_news (const QString& news);
   void open_bug_tracker_page (void);
   void open_octave_packages_page (void);
   void open_agora_page (void);
   void open_contribute_page (void);
-  void open_developer_page (void);
+  void open_donate_page (void);
   void process_settings_dialog_request (const QString& desired_tab
                                                          = QString ());
 
   void copy_image_to_clipboard (const QString& file, bool remove_file);
 
   void show_about_octave (void);
   void notice_settings (const QSettings *settings);
   void confirm_shutdown_octave (void);
